Sources/KituraContracts/Contracts.swift

```diff
@@ -22,7 +22,7 @@
  and server side (eg Kitura) of the request (typically a HTTP REST request).
 
  ### Usage Example: ###
- ````
+ ````swift
```

Comment:
// MARK: Accessing information about the error.
Please can you remove the full stop from the above line as all other sections don't have full stops after them so it looks inconsistent.